include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -187,13 +187,15 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Normalize,Clamp,Sw
 					};
 				};
 				
-				uint32_t				inMipLevel = 0u;
-				uint32_t				outMipLevel = 0u;
-				ICPUImage*				inImage = nullptr;
-				ICPUImage*				outImage = nullptr;
-				KernelX					kernelX;
-				KernelY					kernelY;
-				KernelZ					kernelZ;
+				uint32_t							inMipLevel = 0u;
+				uint32_t							outMipLevel = 0u;
+				ICPUImage*							inImage = nullptr;
+				ICPUImage*							outImage = nullptr;
+				KernelX								kernelX;
+				KernelY								kernelY;
+				KernelZ								kernelZ;
+				core::smart_refctd_ptr<ICPUBuffer>	phaseSupportLUT = nullptr;
```

Comment:
store the LUT in some part of the state's `scratch` (its for storing everything, really)

Reply:
Resolved.